apps/web/src/utils/profanity.ts

```diff
@@ -0,0 +1,9 @@
+import Filter from 'bad-words';
+import overrideWords from './bad-word-overrides.json';
+
+const f = new Filter();
+
+export function containsProfanity(str: string): boolean {
+  f.addWords(...overrideWords);
```

Comment:
cant this be moved outside the functyion as well?

Reply:
yeah, I didn't notice that. will fix